File: app/src/main/java/com/projetos/redes/modelos/ConsumoInternet.java
package com.projetos.redes.modelos;

import com.projetos.redes.Utils;
import com.projetos.redes.utilidades.UtilidadeData;

public class ConsumoInternet {

    private long wifi, mobile, data_inicio, data_fim;

    public ConsumoInternet(long wifi, long mobile, long inicio, long fim) {
        this.wifi = wifi;
        this.mobile = mobile;
        this.data_inicio = inicio;
        this.data_fim = fim;
    }

    public void setWifi(long wifi) {
        this.wifi = wifi;
    }

    public void setMobile(long mobile) {
        this.mobile = mobile;
    }

    public long getWifi() {
        return wifi;
    }

    public long getMobile() {
        return mobile;
    }

    public long getDataInicio() {
        return data_inicio;
    }

    public long getDataFim() {
        return data_fim;
    }

    public long getTotal(){
        return wifi + mobile;
    }

    public String toString(){
        return String.format("Wi-Fi: %s\nMobile: %s\nInicio: %s\nFim: %s",
                Utils.converterParaMegabytes(wifi), Utils.converterParaMegabytes(mobile),
                UtilidadeData.toDateString(data_inicio), UtilidadeData.toDateString(data_fim));
    }

}

File: modelo/src/main/java/com/projetos/redes/database/modelos/ConsumoInternet.kt
package com.projetos.redes.database.modelos

import androidx.room.Entity
import androidx.room.PrimaryKey
import java.time.LocalDateTime

@Entity(tableName = "consumo_internet")
data class ConsumoInternet(
            @PrimaryKey(autoGenerate = true) val id: Int,
            val wifi: Long ,
            val mobile: Long,
            val dataInicio: LocalDateTime? = LocalDateTime.now(),
            val dataFim: LocalDateTime? = LocalDateTime.now())